submissions/andysmokk/oop-exercise/index.js

```diff
@@ -0,0 +1,120 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/andysmokk/a-tiny-JS-world
+   Web app: https://andysmokk.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, legs, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.legs = legs;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  showPropertiesInhabitant() {
+    const propertiesInhabitant = [
+      this.species,
+      this.name,
+      this.legs,
+      this.gender,
+      this.saying,
+      this.friends,
+    ];
+
+    return propertiesInhabitant.map((property) =>
+      property === this.name
+        ? `<strong>${property}</strong>`
+        : property === this.saying
+        ? `<em>"${property}"</em>`
+        : Array.isArray(property)
+        ? property.join(", ")
+        : property
```

Comment:
Why building up a complex nested ternary logic instead of just converting properties into desired form when declaring the source array (`propertiesInhabitant`)?

Reply:
I just like ternary operator:grin:. Fixed.

Reply:
When you start putting nested ternaries of multiple `if ... else if ... else if ... else` think of taking different appraoches.

Reply:
I agree with you. Thanks for the explanation and your work.